src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -202,6 +231,39 @@ const styles = StyleSheet.create({
     height: '100%',
     justifyContent: 'center'
     // backgroundColor: 'blue'
+  },
+  buttonBlockUser: {
+    width: 88,
+    height: 36,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 8,
+    borderColor: colors.bondi_blue,
+    borderWidth: 1
+  },
+  textButtonBlock: {
+    fontFamily: fonts.inter[600],
+    fontWeight: 'bold',
+    fontSize: 12,
+    color: colors.white
+  },
+  textButtonBlockUser: {
+    fontFamily: fonts.inter[600],
+    fontWeight: 'bold',
+    fontSize: 12,
+    color: colors.bondi_blue
+  },
+  buttonBlock: {
+    width: 88,
+    height: 36,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: 1,
+    borderColor: colors.blockColor,
+    borderRadius: 8,
+    backgroundColor: colors.blockColor
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for the block/unblock buttons. Ensure that these styles align with the overall design language of your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->